src/lib/components/Accordion/AccordionItem.js

```diff
@@ -3,17 +3,13 @@ import PropTypes from 'prop-types';
 import './Accordion.css';
 
 class AccordionItem extends React.Component {
-  constructor() {
-    super();
-    this.accordionItemToggle = this.accordionItemToggle.bind(this);
-    this.state = {
-      hidden: true,
-    };
+  constructor(props) {
```

Comment:
On previous code reviews, Ant and I agreed we would not pass props to constructors if they were not being used. I know it says the opposite in the React docs but we agreed with: https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e